srcpkgs/lc0/template

```diff
@@ -0,0 +1,39 @@
+# Template file for 'lc0'
+pkgname=lc0
+version=0.27.0
+revision=1
+wrksrc="lc0-${version}"
+build_style=meson
+configure_args="-Dgtest=false --buildtype=release"
+meson_builddir="build/release"
+_githash=00fd892e648160c294346c87449126d9bad80a16
+hostmakedepends="tar pkg-config"
+makedepends="$(vopt_if openblas openblas-devel)
+ $(vopt_if opencl ocl-icd-devel) protobuf-devel zlib-devel eigen"
+short_desc="UCI chess engine designed to play via neural network"
+maintainer="shtayerc <david.murko@mailbox.org>"
+license="GPL-3.0-only"
+homepage="http://lczero.org"
+distfiles="https://github.com/LeelaChessZero/lc0/archive/v${version}.tar.gz
+ https://github.com/LeelaChessZero/lczero-common/archive/${_githash}.tar.gz"
+checksum="2f38cd5fb1822798d10d806fe8930f60a5b3a7f92b9dfcbbeb210df8a8b37b86
+ c3de32a249b25f092218b7550ce00134998cf3fef8ebbb08801883ed48f36576"
+
+build_options="openblas opencl"
+build_options_default="openblas"
+
+if [ "$CROSS_BUILD" ] && [ "$build_option_openblas" ]; then
+	configure_args+=" -Dopenblas_include=${XBPS_CROSS_BASE}/usr/include/openblas/"
+fi
+
+if [ "$XBPS_TARGET_NO_ATOMIC8" ]; then
+	makedepends+=" libatomic-devel"
+fi
+
+post_extract() {
+	cp -a ../lczero-common-${_githash}/proto/ ./libs/lczero-common/
```

Comment:
Please use some form of `mv` so the directory isn't left after the build is done. something like `rmdir libs/lczero-common` then `mv ../lczero-common-${_githash} libs/lczero-common` should work.

Also, consider being specific: `_lczero_commit`